datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -737,6 +737,7 @@ async fn test_physical_plan_display_indent_multi_children() {
 }
 
 #[tokio::test]
+#[cfg_attr(tarpaulin, ignore)]
```

Comment:
These 4 tests run fail in `coverage`(don't know why 🤕 ), so i try to ignore them in coverage, but they still check in `cargo test`.

Reply:
How did they fail?